docs/src/main/paradox/routing-dsl/directives/security-directives/authenticateBasic.md

```diff
@@ -23,7 +23,7 @@ Longer-running authentication tasks (like looking up credentials in a database)
 variant of this directive which allows it to run without blocking routing layer of Akka HTTP, freeing it for other requests.
 
 Standard HTTP-based authentication which uses the `WWW-Authenticate` header containing challenge data and
-@unidoc[Authorization] header for receiving credentials is implemented in subclasses of `HttpAuthenticator`.
+@unidoc[Authorization] header for receiving credentials is implemented in `extractCredentialsAndAuthenticateOrRejectWithChallenge`.
```

Comment:
That is an internal method, it should be `authenticateOrRejectWithChallenge` which is the public API directive

Reply:
my bad! I didn't realize that. updated now.